components/code-editor/index.js

```diff
@@ -102,4 +183,29 @@ class CodeEditor extends Component {
 	}
 }
 
-export default CodeEditor;
+export default withLazyDependencies( {
```

Comment:
What do you think about calling it `withLazyLoadedAssets`?

Reply:
I like 'dependency' since it's [what we call them in Core](https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/class-wp-dependency.php#L19).

Happy to be persuaded though 🙂